### PR TITLE
colored the link and file-path

### PR DESCRIPTION
**Pull request by 258204**

### CODE REVIEW

**Comment by CleanMachine1** [comment]
I have no reason why however these string(color...) seem to leave a space whereever they are, thats why I tend to "declare" them on a new line so it doesn't add the space.

`Saving to  /home/$USER/Git/tldr-page-creator/git.md  and exiting.`

Notice how there is a double space..

I don't have a method to fix this since using printf doesn't work with string(color....)

File: tldr-page.go [hunk 1]
```diff
@@ -151,7 +151,8 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	fmt.Println("Saving to " + path + "/" + pagename + " and exiting.")
-	fmt.Println("\nIf you want to contribute this page to TLDR, please follow the instructions ")
-	fmt.Println("from the following link: https://github.com/tldr-pages/tldr#how-do-i-contribute")
+	fmt.Println("Saving to", string(colorBlue), path + "/" + pagename ,string(colorWhite), "and exiting.")
+	fmt.Println("If you want to contribute this page to TLDR, please follow the instructions ")
+	fmt.Println("from the following link:")
+	fmt.Print(string(colorBlue), "https://github.com/tldr-pages/tldr#how-do-i-contribute\n",string(colorWhite))
```
Comment (CleanMachine1) [inline tldr-page.go]:
```suggestion
	fmt.Print(string(colorBlue), "https://github.com/tldr-pages/tldr#how-do-i-contribute\n", string(colorWhite))
```

**Comment by 258204** [comment]
Fixed the double spaces problem.

**Comment by CleanMachine1** [comment]
I just cut down some of the lines, this also removes the double spaces.

```go
	fmt.Print("Saving to ", string(colorBlue), path+"/"+pagename, string(colorWhite), " and exiting.\n")
	fmt.Println("If you want to contribute this page to TLDR, please follow the instructions\nfrom the following link:")
	fmt.Print(string(colorBlue), "https://github.com/tldr-pages/tldr#how-do-i-contribute\n", string(colorWhite))
```

**Comment by 258204** [comment]
Nice, just added the suggestion.

**Comment by CleanMachine1** [comment]
Thank you very much!

**Pull request merged**